CMakeLists.txt

```diff
@@ -106,33 +106,33 @@ set_target_properties(${PROJECT_NAME}_filter PROPERTIES COMPILE_FLAGS "${CMAKE_C
 set_target_properties(${PROJECT_NAME}_filter PROPERTIES LINK_FLAGS "${OpenMP_CXX_FLAGS}")
 
 # Demo filter executable
-add_executable(${PROJECT_NAME}_filter_test src/demo/grasp_filter_test.cpp)
-target_link_libraries(${PROJECT_NAME}_filter_test
+add_executable(${PROJECT_NAME}_grasp_filter_demo src/demo/grasp_filter_demo.cpp)
+target_link_libraries(${PROJECT_NAME}_grasp_filter_demo
```

Comment:
please split this file into both a test and a demo. it already has a unit test in it - called ``unitTest()``, but it was never finished - it just needs gtest added from our template. the tests are already there:

Reply:
see reply regarding tests